keras_cv/models/backbones/test_backbone_presets.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Backbone presets for tests. Includes smallest preset for each backbone."""
+
+test_backbone_presets = [
```

Comment:
Why not add this to `backbone_presets.py`?

Reply:
Just to keep the test code separate from the non-test code. 🤷